com.obsproject.Studio.Plugin.obs-kinect.yaml

```diff
@@ -0,0 +1,75 @@
+id: com.obsproject.Studio.Plugin.obs-kinect
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.3
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/obs-kinect
+  env:
+    PKG_CONFIG_PATH: /app/plugins/obs-kinect/lib/pkgconfig #needed for libfreenect2
+modules:
+  - shared-modules/libusb/libusb.json
+
+#  - name: glfw
+#    buildsystem: cmake
+#    config-opts:
+#      - -DBUILD_SHARED_LIBS=ON
+#    sources:
+#      - type: archive
+#        url: https://github.com/glfw/glfw/releases/download/3.3.8/glfw-3.3.8.zip
+#        sha256: 4d025083cc4a3dd1f91ab9b9ba4f5807193823e565a5bcf4be202669d9911ea6
+
+#  - name: libjpeg-turbo
+#    buildsystem: cmake
+#    config-opts:
+#      - -DCMAKE_INSTALL_LIBDIR=/app/plugins/obs-kinect/lib
+#    sources:
+#      - type: archive
+#        url: https://github.com/libjpeg-turbo/libjpeg-turbo/archive/refs/tags/2.1.4.tar.gz
+#        sha256: a78b05c0d8427a90eb5b4eb08af25309770c8379592bb0b8a863373128e6143f
+
+  - name: libfreenect
+    buildsystem: cmake
```

Comment:
```suggestion
    buildsystem: cmake-ninja
```

Reply:
I'm not sure what are the differences, but it does build slightly faster with ninja.